deploy/kubernetes/images/stable-master/image.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: imagetag-csi-provisioner
 imageTag:
   name: k8s.gcr.io/sig-storage/csi-provisioner
-  newTag: "v2.2.1"
```

Comment:
Update the images in the alpha or dev overlay first, and let it run through the postsubmits, we'll update stablemaster only after it's validated.

Reply:
done